nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -427,7 +456,8 @@ def __init__(self, target_model: NNCFNetwork,
                  weights_normalizer_fn: Optional[Callable[[torch.Tensor], torch.Tensor]],
                  node_name_vs_dynamic_input_width_op_map: Dict[NNCFNodeName, ElasticWidthOp],
                  pruned_module_groups_info: Clusterization[ElasticWidthInfo](id_fn=lambda x: x.node_name),
-                 transformation_commands: List[TransformationCommand]):
+                 transformation_commands: List[TransformationCommand],
+                 add_dynamic_inputs = None):
```

Comment:
Is this variable type `List[str]` or `Optional[List[str]]`?

Reply:
@vinnamkim. Fixed. Thanks!